src/fontMetrics.js

```diff
@@ -1,6 +1,5 @@
 /* eslint no-unused-vars:0 */
 
-const Style = require("./Style");
```

Comment:
This was an unused cyclic import.  Style.js requires fontMetrics.js.  It was picked up when switching to jest.